test/command/suite/tokenize/without_attributes.test

```diff
@@ -0,0 +1,6 @@
+#@require-feature mecab
+
+tokenize \
+  --tokenizer TokenMecab \
+  --string 'Groongaはオープンソースのカラムストア機能付き全文検索エンジンです。' \
+  --without_attributes
```

Comment:
In general, we should not use negative word ("without" in this case) for option name. It's difficult to understand.

See also: https://www.clear-code.com/blog/2014/2/6.html